src/Makefile

```diff
@@ -1,10 +1,12 @@
-GCC_FLAGS=-c -Wall -Werror -g
-CLANG_FLAGS=-c -Wall -Werror -g
+GCC_FLAGS=-c -Wall -Werror -g -fPIC 
+CLANG_FLAGS=-c -Wall -Werror -g -fPIC 
+GCC_SO_FLAGS= -shared -Wl,-soname,libgherkin.so.1
```

Comment:
".1"? not ".1.0" as everywhere else?

Reply:
Or better - take the version from a `VERSION` file as described above